src/worker.rs

```diff
@@ -149,6 +162,21 @@ impl Worker {
                         } else {
                             None
                         };
+                        let resp = if mutated && allowed_to_mutate.as_ref() == Some(&false) {
+                            ValidationResponse {
+                                allowed: false,
+                                status: Some(ValidationResponseStatus {
+                                    message: Some(format!("Request rejected by policy {}. The policy attempted to mutate the request, but it is currently configured to not allow mutations.", &req.policy_id)),
+                                    code: None,
+                                }),
+                                // if the mutate is false, we are in a validating webhook. If we send a patch, k8s will failed. Because validating webhook should not has patch
```

Comment:
grammar fix:

```suggestion
                                // if `allowed_to_mutate` is false, we are in a validating webhook. If we send a patch, k8s will fail because validating webhook do not expect this field
```